core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7716BuildDeadlock.java

```diff
@@ -0,0 +1,68 @@
+package org.apache.maven.it;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.io.File;
+
+import org.apache.maven.shared.verifier.Verifier;
+import org.apache.maven.shared.verifier.util.ResourceExtractor;
+import org.junit.jupiter.api.Test;
+
+/**
+ * This is a test set for
+ * <a href="https://issues.apache.org/jira/browse/MNG-7716">MNG-7716</a>.
+ * Executing the project should not deadlock
+ *
+ */
+class MavenITmng7716BuildDeadlock
+    extends AbstractMavenIntegrationTestCase
+{
+
+    public MavenITmng7716BuildDeadlock()
+    {
+        super( "[3.8.8,)[3.9.1,)[4.0.0-alpha-5,)"  );
```

Comment:
I don't this that this is correct because this test does not apply to 3.9.0:
https://github.com/apache/maven-integration-testing/commit/5dd08b8931e3ceecc6fb2621c4d8214683b12920#diff-f7e71461bbf4f4479ee33d4aab9581ade83aa096bd93907acba4d337e3b760d6R40

@cstamas WDYT?

Reply:
3.9.1 is the first included version...

Reply:
But `[3.8.8,)` will match 3.9.0 as well which is not correct.

Reply:
Adjusted!